bin/ledger2beancount

```diff
@@ -465,12 +473,14 @@ sub map_account($) {
 sub map_commodity($) {
     my ($commodity) = @_;
 
+    $ledger_commodities{$commodity} = 1;
+
     $commodity = $config->{commodity_map}{$commodity} if exists $config->{commodity_map}{$commodity};
     $commodity =~ s/(^")|("$)//g;
     # Check again after removing the quote
     $commodity = $config->{commodity_map}{$commodity} if exists $config->{commodity_map}{$commodity};
 
-    # Maxium limit for beancount commodities: 24 characters
+    # Maximum limit for beancount commodities: 24 characters
```

Comment:
Can we make this a constant at the beginning of the file? It will avoid both writing it twice and having magic numbers in the code (feel free to push a fix for this directly to master)